tools/versions.md

```diff
@@ -39,41 +41,46 @@ How to read the columns:
 ||5|EOL ❌|||||D|2020-09-30|2021-12 with Dome|
 ||6|EOL ❌|||F||E|2020-03-31|2022-03 with Edifice|
 ||7|stable|J K L|B F J|F J||F|2021-09-30|2026-09 with Fortress|
-||8|stable||F J|||G|2022-09-26|2024-11 with Garden|
-||9|stable||J|||H|2022-09-26|TBD|
+||8|EOL ❌||F J|||G|2022-09-26|2024-11 with Garden|
+||9|stable||J N|||H|2023-09-29|2028-09 with Harmonic|
```

Comment:
should the EOL month be `08`?

Reply:
hmm, I just copied the EOL dates from https://gazebosim.org/docs/latest/releases/.

Reply:
oh because I see `2028-08 with Harmonic` in other places in this PR. So we should change all of them to `2028-09 with Harmonic`

Reply:
Fixed in https://github.com/gazebosim/docs/pull/549/commits/62a82d0664fa7b8b463535045cfc0f134ab69687